src/compiler/checker.ts

```diff
@@ -8475,11 +8467,11 @@ namespace ts {
                         return type;
                     }
                     const doubleEquals = operator === SyntaxKind.EqualsEqualsToken || operator === SyntaxKind.ExclamationEqualsToken;
-                    const facts = doubleEquals ?
-                        assumeTrue ? TypeFacts.EQUndefinedOrNull : TypeFacts.NEUndefinedOrNull :
-                        value.kind === SyntaxKind.NullKeyword ?
-                            assumeTrue ? TypeFacts.EQNull : TypeFacts.NENull :
-                            assumeTrue ? TypeFacts.EQUndefined : TypeFacts.NEUndefined;
+                    const facts = doubleEquals
+                        ? assumeTrue ? TypeFacts.EQUndefinedOrNull : TypeFacts.NEUndefinedOrNull
+                        : value.kind === SyntaxKind.NullKeyword
+                            ? assumeTrue ? TypeFacts.EQNull : TypeFacts.NENull
+                            : assumeTrue ? TypeFacts.EQUndefined : TypeFacts.NEUndefined;

```

Comment:
Not a fan of the `?` and `:` prefix style. Would prefer to keep the existing formatting.